examples/tensorflow-backdoor-poisoning/demo-mnist-poison-backdoor-adv-embedding.ipynb

```diff
@@ -79,6 +79,9 @@
     "    else f\"http://localhost:{MLFLOW_TRACKING_PORT}\"\n",
     ")\n",
     "\n",
+    "# Path to custom task plugins archives\n",
+    "CUSTOM_PLUGINS_EVALUATION_TAR_GZ = Path(\"custom-plugins-evaluation.tar.gz\")\n",
```

Comment:
If you change the name of the generated tar file in the Makefile, it will need to be updated here to match.